tests/integration/targets/icinga/runme.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+set -eux
+
+ansible-playbook icinga_agent.yml
+ansible-playbook icinga_plugins.yml
```

Comment:
should we also include 2 additional "--check --diff" executes?
`ansible-playbook icinga_agent.yml --check --diff
ansible-playbook icinga_plugins.yml --check --diff
ansible-playbook icinga_agent.yml
ansible-playbook icinga_plugins.yml`